webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/reports/CachedFindRuleNodeStatusReportsTest.scala

```diff
@@ -129,26 +132,9 @@ class CachedFindRuleNodeStatusReportsTest extends Specification {
     )
   )
 
-  object testNodeInfoService extends NodeInfoService {
-    def getNodeInfo(nodeId: NodeId):           IOResult[Option[NodeInfo]]      = ???
-    def getNodeInfoPure(nodeId: NodeId):       IOResult[Option[NodeInfo]]      = ???
-    def getNodeInfos(nodesId: Set[NodeId]):    IOResult[Set[NodeInfo]]         = ???
-    def getNodeInfosSeq(nodeIds: Seq[NodeId]): IOResult[Seq[NodeInfo]]         = ???
-    def getNode(nodeId: NodeId):               Box[Node]                       = ???
-    def getAllNodes():                         IOResult[Map[NodeId, Node]]     = ???
-    def getAllNodeInfos():                     IOResult[Seq[NodeInfo]]         = ???
-    def getAllNodesIds():                      IOResult[Set[NodeId]]           = ???
-    def getAllSystemNodeIds():                 IOResult[Seq[NodeId]]           = ???
-    def getPendingNodeInfos():                 IOResult[Map[NodeId, NodeInfo]] = ???
-    def getPendingNodeInfo(nodeId: NodeId):    IOResult[Option[NodeInfo]]      = ???
-    def getDeletedNodeInfos():                 IOResult[Map[NodeId, NodeInfo]] = ???
-    def getDeletedNodeInfo(nodeId: NodeId):    IOResult[Option[NodeInfo]]      = ???
-    def getNumberOfManagedNodes:               IOResult[Int]                   = ???
-    val getAll:                                IOResult[Map[NodeId, NodeInfo]] = {
-      nodes.map { case (n, _, _) => (n.id, n) }.toMap.succeed
-    }
-
-  }
+  val accepted     = nodes.map { case (n, _, _) => n }.toMap
+  val nodeFactRepo =
+    CoreNodeFactRepository.make(NoopFactStorage, NoopGetNodesbySofwareName, Map(), accepted, Chunk.empty).runNow
```

Comment:
Now the base repo for `CachedFindRuleNodeStatusReportsTest` is a `NodeFactRepository`